pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -248,16 +253,24 @@ func CreateGCECloud(apiEndpoint, projectID, region, zone string, managedZones []
 		return nil, err
 	}
 
-	if apiEndpoint != "" {
-		service.BasePath = fmt.Sprintf("%sprojects/", apiEndpoint)
+	client, err = newOauthClient(tokenSource)
+	serviceBeta, err := computebeta.New(client)
+	if err != nil {
+		return nil, err
 	}
 
 	client, err = newOauthClient(tokenSource)
-	serviceBeta, err := computebeta.New(client)
+	serviceAlpha, err := computealpha.New(client)
 	if err != nil {
 		return nil, err
 	}
 
+	if apiEndpoint != "" {
+		service.BasePath = fmt.Sprintf("%sprojects/", apiEndpoint)
+		serviceBeta.BasePath = fmt.Sprintf("%sprojects/", strings.Replace(apiEndpoint, "v1", "beta", 0))
+		serviceAlpha.BasePath = fmt.Sprintf("%sprojects/", strings.Replace(apiEndpoint, "v1", "alpha", 0))
```

Comment:
It seems like it:
prod: https://www.googleapis.com/compute/v1/
stage: https://www.googleapis.com/compute/staging_v1/
devcluster: http://localhost:3990/compute/v1